pkg/generators/forwarding/fluentd/output_conf_loki_test.go

```diff
@@ -0,0 +1,82 @@
+package fluentd
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1alpha1"
+	test "github.com/openshift/cluster-logging-operator/test"
+)
+
+var _ = Describe("Generating external loki server output store config block", func() {
+	var (
+		err       error
+		outputs   []v1alpha1.OutputSpec
+		generator *ConfigGenerator
+	)
+	BeforeEach(func() {
+		generator, err = NewConfigGenerator(false, false)
+		Expect(err).To(BeNil())
+	})
+
+	Context("for a loki endpoint", func() {
+		lokiConf := `<label @LOKI_RECEIVER>
+        <match **>
+            @type loki
+            url https://logs-us-west1.grafana.net
+            tenant a-tenant
+            line_format json
+            extract_kubernetes_labels true
+            remove_keys docker, kubernetes, pipeline_metadata, hostname, index_name
+            <label>
+                # docker
+                container_id $.docker.container_id
+
+                # kubernetes
+                container_name $.kubernetes.container_name
+                container_image $.kubernetes.container_image
+                container_image_id $.kubernetes.container_image_id
+                kubernetes_host $.kubernetes.host
+                master_url $.kubernetes.master_url
+                pod_name $.kubernetes.pod_name
+                pod_id $.kubernetes.pod_id
+                namespace_name $.kubernetes.namespace_name
+                namespace_id $.kubernetes.namespace_id
+
+                # general
+                index_name $.viaq_index_name
+                hostname $.hostname
+            </label>
+            <buffer>
+                @type file
+                path '/var/lib/fluentd/loki_receiver'
+                flush_interval "#{ENV['ES_FLUSH_INTERVAL'] || '1s'}"
```

Comment:
I would set these to a value and skip the ability to utilize an ENV var.  This is a pattern left over from when you could modify the env var and their was no operator

Reply:
All `ES_` prefixed env vars are removed. I kept the `BUFFER_` prefixed env vars which are set in run.sh